### PR TITLE
Add --cacert flag to the installer

### PR DESCRIPTION
**Pull request by mansam**
Fixes https://github.com/vmware-tanzu/velero/issues/2333

Allows setting the cacert field on the BSL during
the install process using the file at the path
specified by the --cacert field.

Signed-off-by: Sam Lucidi <slucidi@redhat.com>

### CODE REVIEW

File: pkg/cmd/cli/install/install.go [hunk 1]
```diff
@@ -99,6 +100,7 @@ func (o *InstallOptions) BindFlags(flags *pflag.FlagSet) {
 	flags.DurationVar(&o.DefaultResticMaintenanceFrequency, "default-restic-prune-frequency", o.DefaultResticMaintenanceFrequency, "how often 'restic prune' is run for restic repositories by default. Optional.")
 	flags.Var(&o.Plugins, "plugins", "Plugin container images to install into the Velero Deployment")
 	flags.BoolVar(&o.CRDsOnly, "crds-only", o.CRDsOnly, "only generate CustomResourceDefinition resources. Useful for updating CRDs for an existing Velero install.")
+	flags.StringVar(&o.CACert, "cacert", o.CACert, "path to a certificate bundle to use when verifying TLS connections to the object store")
```
Comment (skriss) [inline pkg/cmd/cli/install/install.go]: please add "Optional." to the end of the flag description

Comment (carlisia) [reply pkg/cmd/cli/install/install.go]: If you don't mind, please use the same language as for the secret (ie `file containing...`) here, or change it there to be `path to...`. Thanks.

Comment (carlisia) [reply pkg/cmd/cli/install/install.go]: And maybe `CACert` could be `CACertFile`.

**Comment by skriss** [comment]
thanks again @mansam!

**Pull request merged**